src/host/popup.cpp

```diff
@@ -32,7 +32,7 @@ Popup::Popup(SCREEN_INFORMATION& screenInfo, const COORD proposedSize) :
     _screenInfo(screenInfo),
     _userInputFunction(&Popup::_getUserInputInternal)
 {
-    _attributes = screenInfo.GetPopupAttributes()->GetLegacyAttributes();
+    _attributes = *screenInfo.GetPopupAttributes();

```

Comment:
This is the only one I don't understand, but I trust that it's necessary.

Reply:
PopupAttrs returns a pointer. We were needlessly converting it to a legacy attribute here, now we can just set it without converting to legacy.

Reply:
Note that it's also needlessly a pointer. That's one of the things I ended up cleaning up #6358, but you can do that here if you prefer.

Reply:
I'll leave that one in 6358; i'd like this to be 99% mechanical and not change too many call sites :smile:

thanks!